extensions/example-extension/renderer.tsx

```diff
@@ -30,13 +55,48 @@ export default class ExampleExtension extends LensRendererExtension {
     }
   ];
 
-  clusterPageMenus: Interface.PageMenuRegistration[] = [
+  clusterPageMenus: Interface.ClusterPageMenuRegistration[] = [
```

Comment:
With `clusterPageMenus` typed as `Interface.PageMenuRegistration[]` the `id` and `parentId` fields below caused a ts compile error. But declaring it as `Interface.ClusterPageMenuRegistration[]` results in the foldout menu not getting added to the sidebar. Regular `clusterPageMenus` appear.

Reply:
Tried to use `Interface.ClusterPageMenuRegistration` in my PR, works fine without any compilation errors.
Try to remove `extensions/example-extension/node_modules` and restart app?